projects/canopy/src/lib/forms/checkbox-group/checkbox-group.component.ts

```diff
@@ -102,21 +102,23 @@ export class LgCheckboxGroupComponent implements ControlValueAccessor {
     this._validationElement = element;
   }
 
-  _value: Array<string> = null;
+  private _value: Array<string> = [];
   @Input()
   get value() {
     return this._value;
   }
-  set value(value) {
+  set value(value: Array<string>) {
     this._value = value;
     this.onChange(value);
-    if (this.checkboxes) {
-      this.checkboxes.forEach((checkbox) => {
-        if (value.includes(checkbox.value.toString())) {
-          checkbox.checked = true;
-        }
-      });
+
+    if (!this.checkboxes) {
+      return;
     }
+
+    this.checkboxes.forEach(
+      (checkbox) =>
+        (checkbox.checked = value.includes(checkbox.value.toString()) ? true : false),
```

Comment:
do we need the ternary expression here?

`(method) Array<string>.includes(searchElement: string, fromIndex?: number): boolean`

Reply:
ah good spot, yes we can simplify that